python_modules/dagster/dagster/_cli/blueprint.py

```diff
@@ -0,0 +1,97 @@
+import json
+from importlib import import_module
+from pathlib import Path
+from typing import Sequence
+
+import click
+
+from dagster import _check as check
+from dagster._core.blueprints.load_from_yaml import YamlBlueprintsLoader
+from dagster._core.definitions.load_assets_from_modules import find_objects_in_module_of_types
```

Comment:
Looks like the behavior here is slightly distinct from loading a Dagster code location, where we only care about the top-level `defs`, if present, ignoring all other top-level objects.

Will the loaders end up getting passed directly to `Definitions`? (e.g. do the `Definitions` objects they produce retain a reference to the `YamlBlueprintsLoader` which produced them?) If so, it might be nice to have similar "defs-as-entrypoint" behavior for consistency's sake. That way you could define `YamlBlueprintsLoaders` in submodules and not worry about having to import them for the sole reason of making them visible to the CLI.

Reply:
> Will the loaders end up getting passed directly to Definitions?

@benpankow and I chatted about this in person at length, but for external observers: IMO eventually yes. But that means adding a big new concept to our core APIs, so want to punt on it for a bit.